data/blog/2025/2025-02-03-summary-of-the-meeting-in-february-2025.md

```diff
@@ -0,0 +1,62 @@
+---
+id: 2025-02-03-summary-of-the-meeting-in-february-2025
+author: bidi
+title: 'Summary of the meeting in February 2025'
+draft: false
+public: true
+created: '2025-02-04T11:00:00-01:00'
+updated: '2025-02-04T11:00:00-01:00'
+tags:
+    - meeting summary
+---
+
+On Monday, 3 February 2025, the Technical Steering Committee for the Laminas Project held its monthly meeting.
+The topics discussed included:
+
+- Creating a major release for laminas/laminas-session.
+- Setting up email accounts for @getlaminas.org.
+- Dropping Support for PHP 8.1 in future releases for Laminas packages.
+- New design and layout for getlaminas.org.
+
+<!--- EXTENDED -->
+# Creating a major release for [laminas/laminas-session](https://github.com/laminas/laminas-session)
```

Comment:
We do not place links anywhere in the headlines and I would not start with that either.
Use the first occurrence in text and set a link.

Reply:
Use `laminas-session` instead of `laminas/laminas-session` and then it is consistent with the documentation and the website content.